dev/documentation/en/utilities-translation.md

```diff
@@ -24,14 +24,21 @@ msgstr "Cancel"
 msgid "common.cancelled"
 msgstr "Cancelled"
 
-msgid "common.warning"
-msgstr "Warning"
+msgid "common.pendingMessages"
+msgstr[0] "You have one message pending."
+msgstr[1] "You have {$messageCount} messages pending."
+```
+
+The latest example showed us how to setup a pluralized translation for an English locale, other languages however might have more variants according to the quantity, such as Polish, which has 3 variants:
 
-msgid "common.error"
-msgstr "Error"
+```po
+msgid "comments"
+msgstr[0] "%d Komentarz"
+msgstr[1] "%d Komentarze"
+msgstr[2] "%d Komentarzy"
```

Comment:
What do the [0], [1] and [2] variants mean? Or does the coder not need to know about this? Do they just pass the number into `__p()` and gettext does the rest? If so, I think only the translators need to know this and we can remove it from the dev docs here.

Reply:
The `.po` should specify a `plural-forms` expression (e.g. for English it would be `n != 1`), which is responsible to output an index based on the `items`. This way, the meaning depends on the language/expression.

Given that coders need to write at least the initial translations, I think it makes sense to keep at least the English format.

Also, I'm curious if Weblate works out-of-the-box with this simplified format that I've used, perhaps they need an extra property (`msgid_plural`), I'll investigate 😁

Reply:
I just tested some `.po` editors and some of them had problems to recognize it was a plural without the "duplicated" `msgid_plural`, so I've added to avoid unexpected issues.

Reply:
@asmecher do you know if/how we could test if weblate works ok with `msgid_plural`? See the discussion above.